src/test/resources/de/dkfz/roddy/execution/jobs/cluster/lsf/queryExtendedJobStateByIdTest.json

```diff
@@ -0,0 +1,41 @@
+{
+  "COMMAND": "bjobs",
+  "JOBS": 1,
+  "RECORDS": [
+    {
+      "JOBID": "22005",
+      "JOB_NAME": "ls -l",
+      "STAT": "DONE",
+      "USER": "otptest",
+      "QUEUE": "short-dmg",
+      "JOB_DESCRIPTION": "",
+      "PROJ_NAME": "default",
+      "JOB_GROUP": "",
+      "JOB_PRIORITY": "",
+      "PIDS": "46782,46796,46798,46915,47458,47643",
+      "EXIT_CODE": "",
+      "FROM_HOST": "from-host",
+      "EXEC_HOST": "exec-host:exec-host",
+      "SUBMIT_TIME": "Dec 28 19:56",
+      "START_TIME": "Dec 28 19:56",
+      "FINISH_TIME": "Dec 28 19:56 L",
```

Comment:
https://www.ibm.com/support/knowledgecenter/en/SSETD4_9.1.2/lsf_command_ref/bjobs.1.html

L is status information.